TalZivelin/TZivelin-HW-W01.js

```diff
@@ -0,0 +1,125 @@
+1) (Difficulty: mixed) Predict the output of each of the following expressions, and explain the rules which determine it. If the output depends on the value of x, identify the conditions when the expression will output true. Assume the cases are independent, and x is reset to an unknown value before each.
+
+Some of these are tricky! Don't trust your first instinct.
+
+a) "1" == 1 - True because this does not check for the var types.
+
+b) "1" === 1 - Flase, the === checks that both type (string, NUmber) and value are identical.
+
+c) x == 'x' - If X is a number this will be true.
+
+d) x == (x+'') - True, Adding an empty string to a variable will convert it to a string which is true.
+
+e) '' == ' ' -  True, a string with one space is equal to a string with one space
+
+f) x = true - True, x is assigned true, so x is true.
+
+g) var x; x == 'undefined' - False, undefined and 'undefined' are not the same thing.
+
+h) '9'<'10' - False, Javascript reads the first number for comparison so 9 is not smaller than 1.
+
+i) typeof x + 1 === "number" - True if x is a number false if x is a string.
```

Comment:
1i) The order of precedence is important here. The `typeof` function goes before the addition, so it will actually return `"number1"` if `x` is a number.